internal/c/libqb.cpp

```diff
@@ -37148,6 +37148,26 @@ int32 func__resize() {
     return 0;
 }
 
+void sub__windowscale(int32 amount) {
+    int32 tempx, tempy;
+        if (amount < 50) return; // don't scale the screen to less than 50% of it's original size!!

```

Comment:
Is there a reason to restrict the size?

And if we're going to do that, I think it would be good to clip the values to the lowest allowed rather than doing nothing. We could do the same with the `320x240` limit.

Reply:
You have to restrict to some point.  A window can't be scaled to 0% of its size -- it wouldn't exist!  Or a negative percent...  My screen is -320 pixels by -240 pixels.  😱

There comes a point where we seg fault and crash, and it's hard to determine.  You have to keep room for your titlebar's minimize, maximize, close buttons and such.   We could go sans limit, and put a warning there -- user beware -- but I figure anything scaled down beyond 50% probably isn't readable or usable anyway.  

FWIW, _RESIZE sets a 640x480 snapback limit.  Below that, it resets the screen to that size after trying to downscale it.